MapboxNavigation/StyleManager.swift

```diff
@@ -171,6 +174,13 @@ open class StyleManager: NSObject {
         return solar.date.isNighttime(sunrise: sunrise, sunset: sunset) ? .night : .day
     }
     
+    private func postDidApplyStyleNotification(style: Style) {
```

Comment:
another idea was to post the notification from within the style, something like:
```swift
open override func apply() {
    super.apply()
    
    ....
    defer {
        // Post notification
    }
}
```

but this would post the notification before the full style has been applied to the UIAppearance proxy.